server/server.js

```diff
@@ -41,4 +37,52 @@ process.on("unhandledRejection", (err, promise) => {
   closeDBConnection()
 });
 
+const init = () => {
+  const config = new Configuration({
+    apiKey: process.env.OPEN_AI_API_KEY,
+  });
+  return new OpenAIApi(config);
+};
+
+const openai = init();
+const openAiInit = async (openai, prompt) => {
```

Comment:
if you tested it and it works - delete the functions openainit and runprompt, and the execution at line 81